packages/payment-detection/src/near-detector.ts

```diff
@@ -24,16 +25,33 @@ export default class NearNativeTokenPaymentDetector extends ReferenceBasedDetect
     super(advancedLogic.extensions.nativeToken[0], ExtensionTypes.ID.PAYMENT_NETWORK_NATIVE_TOKEN);
   }
 
-  public static getNearContractName = (chainName: string, paymentNetworkVersion = '0.1.0') => {
+  public static getNearContractName = (
+    chainName: string,
+    paymentNetworkVersion = '0.2.0',
+  ): string => {
+    const version = NearNativeTokenPaymentDetector.getVersionOrThrow(paymentNetworkVersion);
+    const versionMap: Record<string, Record<string, string>> = {
+      aurora: { '0.1.0': 'requestnetwork.near', '0.2.0': 'TODO-MISSING' },
```

Comment:
Is this required before merge?

Reply:
The TODO-MISSING

Reply:
Yes thank you, I will have the addresses now